modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -480,6 +480,10 @@ private Model getParent(Model model) {
                     }
                 }
 
+                if (config.getTag() == CodegenType.SERVER) {
```

Comment:
@tsiq-karold Shall we use `.equals()` instead of `==` for string comparision?

Reply:
These aren't strings, `CodegenType` is an enum.

Reply:
Ah you're right. Completely forgot about that.